### PR TITLE
OPSEXP-77: Update and improve documentation - Part 1

### PR DESCRIPTION
**Pull request by gavincornwell**

### CODE REVIEW

File: docs/docker-compose/README.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+| SOLR_SSL_TRUST_STORE_PASSWORD | Password for trust store. See this [page](https://github.com/Alfresco/InsightEngine/blob/master/search-services/README.md#use-alfresco-search-services-docker-image-with-docker-compose) for more details. |  |
+| SOLR_SSL_TRUST_STORE_TYPE | Trust store type. See this [page](https://github.com/Alfresco/InsightEngine/blob/master/search-services/README.md#use-alfresco-search-services-docker-image-with-docker-compose) for more details. | JCEKS |
+| SOLR_SSL_NEED_CLIENT_AUTH | This variable is used to configure SSL (`true` or `false`). See this [page](https://github.com/Alfresco/InsightEngine/blob/master/search-services/README.md#use-alfresco-search-services-docker-image-with-docker-compose) for more details. |  |
+| SOLR_SSL_WANT_CLIENT_AUTH | This variable is used to configure SSL (`true` or `false`). See this [page](https://github.com/Alfresco/InsightEngine/blob/master/search-services/README.md#use-alfresco-search-services-docker-image-with-docker-compose) for more details. |  |
+
+### Alfresco Transform Router
+
```
Comment (montgolfiere) [inline docs/docker-compose/README.md:123]: JFYI: Transform Router is a runnable jar implemented with Spring Boot app. 

Although we provide some env variables in the default ootb application yaml, Spring Boot will also accept a direct override of the underlying property mapping to the dot notation.

Hence, we may wish to add an advanced option at end of the list that links to Spring Boot documentation for externalised configuration, eg.

https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-external-config

File: docs/docker-compose/README.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+| MAX_TRANSFORM_RETRIES |  | 3 |
+| INITIAL_RETRY_TIMEOUT |  | 10000 |
+| INCREASE_RETRY_TIMEOUT |  | 10000 |
+| MAX_IN_MEMORY_SIZE |  Double default limit to 512KiB | 524288 |
+| HOSTNAME |  | "t-router" |
+| CORE_AIO_URL | Transform core AIO URL (in this case the name of the container is used) | "http://transform-core-aio:8090" |
+| FILE_STORE_URL | Shared file store URL (in this case the name of the container is used) | "http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file" |
+
+### Alfresco Transform Core AIO
+
```
Comment (montgolfiere) [inline docs/docker-compose/README.md:156]: As above for Transform Router - runnable jar (Spring Boot app).

File: docs/docker-compose/README.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+| PDFRENDERER_EXE |  | "/usr/bin/alfresco-pdf-renderer" |
+| LIBREOFFICE_HOME |  | "/opt/libreoffice6.3" |
+| IMAGEMAGICK_ROOT |  | "/usr/lib64/ImageMagick-7.0.10" |
+| IMAGEMAGICK_DYN |  | "/usr/lib64/ImageMagick-7.0.10/lib" |
+| IMAGEMAGICK_EXE |  | "/usr/bin/convert" |
+| IMAGEMAGICK_CODERS |  |  |
+| IMAGEMAGICK_CONFIG |  |  |
+
+### Alfresco Shared File Store
+
```
Comment (montgolfiere) [inline docs/docker-compose/README.md:172]: As above for Transform Router - runnable jar (Spring Boot app).

File: docs/docker-compose/README.md (hunk 1 above)
Comment (gavincornwell) [reply docs/docker-compose/README.md:123]: Thanks for your feedback @montgolfiere. To be honest I'm not entirely sure what you mean. 

Are you suggesting we expose all the dot notation properties here too? Are they documented anywhere else? We can catch up on this when we're both back from holiday if that's OK, or feel free to do a separate PR ;-)

File: docs/helm/eks-deployment.md [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+      namespace: alfresco
+    ```
+
+2. Use the kubectl command to create the cluster roles required by the ingress service.
+
+    ```bash
+    kubectl apply -f ingress-rbac.yaml -n alfresco
+    ```
+
+3. Locate the ID of the security
```
Comment (nilsSure) [inline docs/helm/eks-deployment.md:220]: I don't understand this bit.

File: docs/helm/eks-deployment.md [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+    aws eks describe-cluster --name YOUR-CLUSTER-NAME --query "cluster.resourcesVpcConfig.vpcId" --output text
+    ```
+
+3. Find The CIDR range of VPC using the command below (replacing `VPC-ID` with the ID retrieved in the previous step):
+
+    ```bash
+    aws ec2 describe-vpcs --vpc-ids VPC-ID --query "Vpcs[].CidrBlock" --output text
+    ```
+
+4. Go to the [VPC Console](https://console.aws.amazon.com/vpc/home) and search for the VPC using the ID retrieved in step 2 as shown in the screenshot below:
```
Comment (nilsSure) [inline docs/helm/eks-deployment.md:139]: We could add VPC Console -> Security Groups to be more clear

File: docs/helm/eks-deployment.md (hunk 4 above)
Comment (gavincornwell) [reply docs/helm/eks-deployment.md]: Removed this line.

File: docs/helm/eks-deployment.md (hunk 5 above)
Comment (gavincornwell) [reply docs/helm/eks-deployment.md]: Agreed, I have specifically mentioned the security group section and changed the URL to go directly there.

**Comment by amohammedalfresco** [comment]
I believe it would be a good reminder about minimum memory needed to run a reliable docker-compose based environment (like [this](https://github.com/Alfresco/acs-deployment/blob/master/docker-compose/docker-compose.yml#L1), in the docker-compose [README.md](https://github.com/Alfresco/acs-deployment/blob/OPSEXP-77-docs-refresh/README.md).

**Comment by gavincornwell** [comment]
@amohammedalfresco That is a good catch, I have added it to the prereqs section.

**Pull request merged**